felix/dataplane/linux/int_dataplane.go

```diff
@@ -1035,7 +1040,12 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 				config.MaxIPSetSize))
 			dp.RegisterManager(newPolicyManager(rawTableV6, mangleTableV6, filterTableV6, ruleRenderer, 6, config.RulesConfig.NFTables))
 		} else {
-			dp.RegisterManager(newRawEgressPolicyManager(rawTableV6, ruleRenderer, 6, ipSetsV6.SetFilter, config.RulesConfig.NFTables))
+			mgr := newRawEgressPolicyManager(rawTableV6, ruleRenderer, 6, ipSetsV6.MarkDirty, config.RulesConfig.NFTables)
+			ipSetsV6.SetFilter(func(ipSetName string) bool {
```

Comment:
Worth a comment here saying what this filter stuff is about.  
```suggestion
			// When in BPF mode, we still program egress do-not-track rules into
			// (ip|nf)tables; set a filter on the IP sets we program so that only the 
			// IP sets needed for do-not-track are programmed.
			ipSetsV6.SetFilter(func(ipSetName string) bool {
```